schemas/2015-10-01/Microsoft.Media.json

```diff
@@ -0,0 +1,94 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-10-01/microsoft.media.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Media",
+  "description": "Microsoft Media Service Resource Type",
+  "resourceDefinitions": {
+    "mediaServices": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.Media/mediaServices"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-10-01"
+          ]
+        },
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^[a-z0-9]",
+              "minLength": 3,
+              "maxLength": 26
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        },
+        "properties": {
+        "oneOf": [
+          {
+            "$ref": "#/definitions/MediaServiceProperties"
+          },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "properties",
+        "location"
+      ],
+      "description": "Microsoft.Media/mediaServices"
+    }
+  },
+  "definitions" : {
+    "MediaServiceProperties": {
+      "properties": {
+        "apiEndpoints": {

```

Comment:
so the following question is, should apiEndpoints be provided in template? Will service "badRequest" a PUT request which has non-null apiEndpoints property?
If a property should not be provided in PUT request, it should not be in schema.